wagtail/blocks/stream_block.py

```diff
@@ -811,4 +814,6 @@ def media(self):
         )
 
 
+StreamBlock.default_value_class = StreamValue
```

Comment:
I take it this is so others can patch the default value globally?
If so, it could use a comment here.

Reply:
That would be possible, yes. But I would expect that devs would be more likely to subclass `StreamBlock`, then set the attribute on _their version_ of the class, instead of having to specify `value_class`  on init every time. The only reason this attribute is set down here is: `StreamValue` isn't defined until now, and I wanted this to remain a simple attribute.

Reply:
I have changed this to...

```
BaseStreamBlock.default_value_class = StreamValue
```

... as I think it makes a sensible default for all `BaseStreamBlock` subclasses (even if `StreamBlock` is the only one currently)

I have also added a couple of comments [here](https://github.com/wagtail/wagtail/pull/11830/commits/70cd03e821e64ec68b4a6500621c493ccd9a4bab) to clarify the intention.